components/Alerts/alert.html

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./alert.css" />
+    <link rel="stylesheet" href="../Util/util.css" />
+    <link rel="stylesheet" href="../Typography/typography.css" />
+    <link rel="stylesheet" href="../mediaqueries.css" />
+
+    <title>Alerts</title>
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+
+    <title>Document</title>
+  </head>
+  <body>
```

Comment:
I don't see any code snippets or documentation. Do have a look into it. Looks good otherwise.